.env.test

```diff
@@ -28,3 +27,7 @@ GOOGLE_CSE_CX=xxx_google_cse_cx_xxx
 GOOGLE_CSE_KEY=xxx_google_cse_key_xxx
 IMPULSE_APPLICATION_ID=xxx_impulse_id_xxx
 RESIZING_SERVICE=gemini
+
+GRAPHQL_NO_NAME_WARNING=true
+ENABLE_SCHEMA_STITCHING=false
+ENABLE_QUERY_TRACING=false
```

Comment:
@saolsen I’ve disabled these features by default, so this now needs an env variable set to be enabled.